docs/pages/zero-trust-access/rbac-get-started/role-demo.mdx

```diff
@@ -139,13 +167,33 @@ user to access a server with the `env:local-dev` label and deny access to the
 1. Assign the absolute path to the configuration file you created to 
    <Var name="prod-config-path" />.
 
-1. Start the Teleport SSH Service on a local Docker container with the
-   configuration and join it to your cluster:
+1. Start a Docker container on your workstation to prepare a server that you
+   want enroll as a resource in your Teleport cluster:
+   
+   ```code
+   $ docker run -v <Var name="prod-config-path"/>:/etc/teleport.yaml --interactive --tty ubuntu:24.04 /bin/bash
+   ```
+
+1. In your container shell, install curl:
 
    ```code
-   $ docker run -v <Var name="prod-config-path" />:/etc/teleport/teleport.yaml (=teleport.latest_ent_debug_docker_image=)
+   $ apt-get update && apt-get install -y curl
```

Comment:
```suggestion
   $ apt-get update && apt-get install -y curl sudo
```

Not including `sudo` can make user creation fail if another rbac has `sudo` rights.